core/src/main/java/com/datastax/oss/driver/internal/core/loadbalancing/BasicLoadBalancingPolicy.java

```diff
@@ -138,13 +137,10 @@ public BasicLoadBalancingPolicy(@NonNull DriverContext context, @NonNull String
             .getConsistencyLevelRegistry()
             .nameToLevel(profile.getString(DefaultDriverOption.REQUEST_CONSISTENCY));
 
-    Set<String> remoteDcs =
+    preferredRemoteDcs =
         new LinkedHashSet<>(
-            Objects.requireNonNull(
-                profile.getStringList(
-                    DefaultDriverOption.LOAD_BALANCING_DC_FAILOVER_PREFERRED_REMOTE_DCS,
-                    new ArrayList<>())));
-    preferredRemoteDcs = new ArrayList<>(remoteDcs);
+            profile.getStringList(
+                DefaultDriverOption.LOAD_BALANCING_DC_FAILOVER_PREFERRED_REMOTE_DCS));
```

Comment:
DriverExecutionProfile.getStringList(DriverOption) is annotated with NonNull so we don't need to worry about handling the null case here.